05-raster-vector.qmd

```diff
@@ -156,7 +156,7 @@ bb
 ```
 
 The extent can now be used for masking.
-Here, we are also using the `all_touched=True` option so that pixels partially overlapping with the extent are also included in the output.
+Here, we are also using the `all_touched=True` option so that pixels partially overlap with the extent are also included in the output.
```

Comment:
This should be 'overplapping' to keep the original meaning (those pixels which are partially overlapping are *kept*)